src/ec/packages/ec-component-table/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@ecl-twig/ec-component-table",
+  "author": "European Commission",
+  "license": "EUPL-1.1",
+  "version": "2.8.0",
+  "description": "ECL Twig - EC Table",
+  "devDependencies": {
+    "@ecl/ec-component-table": "^2.9.0",
+    "@ecl/ec-specs-table": "^2.9.0",
+    "html-react-parser": "0.9.1"
```

Comment:
do you really need `html-react-parser`? I don't see it used anywhere

Reply:
@yhuard Yes, it is not needed.